extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -287,6 +264,24 @@ export class AmpUserNotification extends AMP.BaseElement {
   }
 
   /**
+   * Returns whether this notification has been dismissed and the dismissal
+   * has been persisted in storage. Returns false if storage throws error or
+   * 'data-persist-dismissal' is disabled.
+   * @return {!Promise<boolean>}
+   */
+  isDismissed() {
+    if (!this.persistDismissal_) {
+      return Promise.resolve(false);
+    }
+    return this.storagePromise_
+        .then(storage => storage.get(this.storageKey_))
+        .then(persistedValue => !!persistedValue, reason => {
```

Comment:
> also prefer .catch for error handler.
 
I told him to combine since `!!value` can never throw, so no reason to create _another_ promise in the chain to catch it and any previous rejections.